Project.toml

```diff
@@ -48,7 +48,7 @@ DocStringExtensions = "0.8"
 DynamicPPL = "0.17.2"
 EllipticalSliceSampling = "0.4"
 ForwardDiff = "0.10.3"
-Libtask = "0.6.7"
+Libtask = "0.7"
```

Comment:
This doesn't have to (and probably shouldn't) be updated in this PR:
```suggestion
```

Reply:
Okay, I just reverted that commit